11-query/04-delete-query.md

```diff
@@ -5,30 +5,35 @@ longTailKeywords: grakn delete data, graql delete query, graql delete instances
 Summary: Delete queries in Grakn.
 ---
 
-## Delete Instances of an Entity Type
-To delete an instance of an entity type from the knowledge graph, we use a [match clause](../11-query/01-match-clause.md) followed by the `delete` keyword and the variable to be deleted. To try the following examples with one of the Grakn clients, follows these [Clients Guide](#clients-guide).
+## Delete Instances of a Type
+To delete an instance of a type from the knowledge graph, we use a [match clause](../11-query/01-match-clause.md) followed by the `delete` keyword and statements indicating data to delete.
```

Comment:
perhaps _**the** statements indicating  **the** data to deleted?

Reply:
I think this makes sense and is grammatical as is!